src/analyze.ts

```diff
@@ -389,8 +389,8 @@ export default async function analyze(id: string, code: string, job: Job): Promi
 
     const wildcardIndex = wildcardRequire.indexOf(WILDCARD);
     const dirIndex = wildcardIndex === -1 ? wildcardRequire.length : wildcardRequire.lastIndexOf(path.sep, wildcardIndex);
-    const wildcardDirPath = wildcardRequire.substr(0, dirIndex);
-    const patternPath = wildcardRequire.substr(dirIndex);
+    const wildcardDirPath = wildcardRequire.substring(0, dirIndex);
+    const patternPath = wildcardRequire.slice(dirIndex);
```

Comment:
Why are some substring and others slice?

Reply:
My intend was to copy the behaviour of the current `substr()`

`slice()` and `substring()` behave differently if the second parameter is negative. 
Examples
```
"test".slice(0, -1) == "tes"
"test".substring(0, -1) == ""
"test".substr(0, -1) == ""
```

Sometimes you want to remove the last x chars in which case `slice()` is what you want but if the second parameter can be positive or negative it's better to use substring* (if the first param is 0, if not it can [depend](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/substring#differences_between_substring_and_slice) as well)